lib/searchmailbox.php

```diff
@@ -21,12 +22,13 @@ class SearchMailbox extends Mailbox {
 	 * @param array $attributes
 	 * @param string $delimiter
 	 */
-	function __construct($conn, $mailBox, $attributes, $delimiter = '/') {
+	function __construct($conn, $mailBox, $attributes, $delimiter='/') {
 		parent::__construct($conn, $mailBox, $attributes, $delimiter);
-		parent::setDisplayName('Favorites');
+		$l = OC::$server->getL10N('mail');
+		parent::setDisplayName((string) $l->t('Favorites'));
 	}
 
-	public function getMessages($from = 0, $count = 2, $filter = '') {
+	public function getMessages($from=0, $count=2, $filter='') {
 		$query = new Horde_Imap_Client_Search_Query();
 		$query->flag('FLAGGED');
```

Comment:
>  @wurstchristoph Can you instead set the flagged attribute to the SearchMailBox object so that the name is translated at the same place as the others : https://github.com/wurstchristoph/mail/blob/translate-favorites/lib/mailbox.php#L263

@zinks- do you mean here? :confused: